pyanaconda/ui/gui/spokes/blivet_gui.py

```diff
@@ -133,6 +127,16 @@ def __init__(self, data, storage, payload):
         StorageCheckHandler.__init__(self)
         NormalSpoke.__init__(self, data, storage, payload)
 
+    @property
+    def category(self):
+        """Category this spoke belongs to."""
+        return SystemCategory
+
+    @property
+    def title(self):
+        """Title of the spoke (will be displayed on the hub)."""
+        return _("GUI|Spoke", "_Blivet-GUI Partitioning")
```

Comment:
Are you sure that  this works? Shouldn't you use `C_`?

Reply:
Oh right, looks like I was a bit to eager in my search and replace - BTW, looking at the spokes in TUI, those don't have a context set. I think we might as well do it now when we are at it ("TUI|Spoke").